packages/app/src/components/Randomizer/Randomizer.js

```diff
@@ -1,62 +1,48 @@
 import React, { useState } from 'react'
 import { Button } from 'react-bootstrap'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons'
 import Filter from '../Filter/Filter/Filter'
+import RestaurantEntry from '../RestaurantEntry/RestaurantEntry'
+import soundService from '../../services/sound'
 import './Randomizer.css'
 
 import restaurantService from '../../services/restaurant'
 
 const Randomizer = () => {
   const [restaurant, setRestaurant] = useState({ name: 'Press the button' })
+  const [disableButton, setDisableButton] = useState(false)
   const [filterType, setFilterType] = useState('some')
   const [filterCategories, setFilterCategories] = useState([])
 
-  const changeRestaurantHandler = async (event) => {
-    event.preventDefault()
-
+  const roll = async (e) => {
+    e.preventDefault()
```

Comment:
We have generally used `event` as event handler event parameter name. Should follow the convention here too.
```suggestion
  const roll = async (event) => {
    event.preventDefault()
```